### PR TITLE
Print memp_fgets and memp_pgs in statreqs

### PR DESCRIPTION
**Pull request by adizaimi**
Print memp_fgets and memp_pgs in statreqs

### CODE REVIEW

File: db/comdb2.c [hunk 1]
```diff
@@ -4559,7 +4559,7 @@ void *statthd(void *p)
                     unsigned nwaits = cur_bdb_stats.n_lock_waits -
                                       last_bdb_stats.n_lock_waits;
                     reqlog_logf(statlogger, REQL_INFO,
-                                "%u locks, avg time %ums, worst time %ums\n",
+                                "%u locks, avg time %ums, worst time=%ums\n",
```
Comment (riverszhang89) [inline db/comdb2.c]: Why do we need the `=` here?

File: db/comdb2.c [hunk 2]
```diff
@@ -4589,6 +4589,16 @@ void *statthd(void *p)
                                     last_bdb_stats.pwrite_time_us) /
                                     npwrites);
                 }
+                if (cur_bdb_stats.n_memp_fgets > last_bdb_stats.n_memp_fgets) {
+                    unsigned n_memp_fgets = cur_bdb_stats.n_memp_fgets - last_bdb_stats.n_memp_fgets;
+                    unsigned us = cur_bdb_stats.memp_fget_time_us - last_bdb_stats.memp_fget_time_us;
+                    reqlog_logf(statlogger, REQL_INFO, "n_memp_fgets=%u, avg time=%ums\n", U2M(us) / n_memp_fgets);
```
Comment (riverszhang89) [inline db/comdb2.c]: The format has 2 arguments, but only 1 is provided? Also maybe rename avg time to something else to not confuse with the one above.

File: db/comdb2.c (hunk 1 above)
Comment (adizaimi) [reply db/comdb2.c]: leftover from other related work--removing. thanks

File: db/comdb2.c (hunk 2 above)
Comment (adizaimi) [reply db/comdb2.c]: Fixing (leftover from other branch). Thanks.

**Pull request merged**